data/timezones.json

```diff
@@ -0,0 +1,29 @@
+[
```

Comment:
Ideally, we'll need to include the places for each corresponding timezone. I added this as a todo to the trello dashboard.